projects/ad9656/common/ad9656_bd.tcl

```diff
@@ -0,0 +1,123 @@
+
+# RX parameters
+set RX_NUM_OF_LANES 4      ; # L
+set RX_NUM_OF_CONVERTERS 4 ; # M
+set RX_SAMPLES_PER_FRAME 1 ; # S
+set RX_SAMPLE_WIDTH 16     ; # N/NP
+
+set RX_SAMPLES_PER_CHANNEL 2 ; # L * 32 / (M * N)
+
+source $ad_hdl_dir/library/jesd204/scripts/jesd204.tcl
+
+# adc peripherals
+
+ad_ip_instance axi_adxcvr axi_ad9656_rx_xcvr
+ad_ip_parameter axi_ad9656_rx_xcvr CONFIG.NUM_OF_LANES $RX_NUM_OF_LANES
+ad_ip_parameter axi_ad9656_rx_xcvr CONFIG.QPLL_ENABLE 0
```

Comment:
Given that it's a single AXI_ADXCVR in the design, I would enable QPLL control

Reply:
QPLL control enabled